src/dlite-entity.c

```diff
@@ -168,7 +168,7 @@ static DLiteInstance *_instance_store_get(const char *id)
   int uuidver;
   char uuid[DLITE_UUID_LENGTH+1];
   DLiteInstance **instp;
-  if ((uuidver = dlite_get_uuid(uuid, id)) != 0 && uuidver != 5)
+  if ((uuidver = dlite_get_uuid(uuid, id)) < 0 || uuidver == 4)
```

Comment:
I don't fully understand the meaning of 4 here. Is it because UUID_RANDOM = 4? If so, then this should be UUID_RANDOM instead.

Reply:
Good point. Fixed